externs/shaka/player.js

```diff
@@ -593,6 +593,7 @@ shaka.extern.DrmConfiguration;
  *   ignoreDrmInfo: boolean,
  *   xlinkFailGracefully: boolean,
  *   ignoreMinBufferTime: boolean,
+ *   ignoreMaxSegmentDuration: boolean,
```

Comment:
You should also modify `lib/util/player_configuration.js` to give this a default value (presumably `false`).